x-pack/platform/plugins/shared/cases/public/components/system_actions/cases/cases_params.test.tsx

```diff
@@ -384,21 +384,6 @@ describe('CasesParamsFields renders', () => {
       expect(await screen.findByTestId('show-time-window-warning')).toBeInTheDocument();
     });
 
-    it('does not update time window if size is less than 5 and unit is in minutes', async () => {
```

Comment:
nit: Could you please update tests here when no error in the beginning and then update time window to 4m and error is shown?
Also similar test for no warning to warning.